docs/pages/app-quickstart.md

```diff
@@ -4,55 +4,26 @@ description: Get started with Zudoku by creating a new Zudoku app using the `cre
 sidebar_icon: app-window-mac
 ---
 
-The recommended way to get started with Zudoku is to use the `create-zudoku-app` CLI tool. This tool will scaffold a new Zudoku site for you to customize and build upon.
-
 # Prerequisites
 
-You will need [Node.js](https://nodejs.org/) installed to use the CLI. Zudoku requires Node.js version 22 or higher.
-
-## Install Zudoku
+- [Node.js](https://nodejs.org/) version 22 or higher
 
-### Windows
+# Getting Started
 
-1. Open your terminal (Command Prompt or PowerShell).
-2. Run the following command to create a new Zudoku app:
+1. Create a new Zudoku app by running:
 
 ```bash
 npx create-zudoku-app
 ```
```

Comment:
````suggestion
```bash
npm create zudoku-app@latest
```
````

Let's use the same ways as on https://zudoku.dev landing page